### ISSUE TITLE
Saved Events: When font size is increased to maximum on iOS, Find Events CTA is off-screen and page can't be scrolled to reveal it

### ISSUE THREAD

**Issue by declanslevin** [redbadger/pride-london-app#212]
## Description of issue

When the font size is increased to the maximum in the Accessibility settings on iOS, the "Find Events" CTA is pushed off-screen by the enlarged text. The page isn't scrollable so you can't reveal it. 

This may be resolved by any work on #98, but logged separately as it is a slightly different issue. 

## Expected behaviour

When the font size is enlarged, the page should be scrollable so that users can still access the Find Events CTA.

## Steps to reproduce

- Make sure you don't have any saved events
- Increase font size to the maximum value (Settings>General>Accessibility>Larger Text)
- Open the "Saved" tab
> CTA is pushed off-screen

## OS / Browser / Device model

iPhone 7+

## Screenshots

<img width=400 src="https://user-images.githubusercontent.com/33623842/39533441-f89f1288-4e26-11e8-873c-2ffb6aec1a96.PNG">

**Comment by declanslevin** [comment]
Fixed!
